Including folding and grouping for ACs

SmithJK opened the pull request:
Including folding and grouping for ACs

SmithJK commented:
This pull request is intentional, but it looks weird... why are none of the files changed? Is it because I changed them earlier (and then reverted)?

SmithJK commented:
Yeah, that failed. Any suggestions? I'm just going to stop for awhile, before I do something *really* stupid.

r3dunlop commented:
Check your state on your machine to make sure you are on the head. Are you pull requesting from the right place? Did you revert and then update locally or something?

> On Apr 8, 2016, at 9:30 PM, SmithJK <notifications@github.com> wrote:
> 
> Yeah, that failed. Any suggestions? I'm just going to stop for awhile, before I do something really stupid.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>

SmithJK commented:
What I did originally:
* changed local version
* pushed to SmithJK:angularcorrelation
* pull request between GRIFFINCollaboration:master and SmithJK:angularcorrelation

What I wanted to do:
* changed local version
* pushed to SmithJK:angularcorrelation
* merge SmithJK:angularcorrelation with SmithJK:master
* pull request between GRIFFIN Collaboration:master and SmithJK:master

Current status:
* SmithJK:master has the correct code (can see it online)
* Pull request between GRIFFINCollaboration:master and SmithJK:master contains no changed files

SmithJK commented:
It looks like GRIFFINCollaboration has a "more recent" version because of the reverted pull request.
